test/clj/rems/api/test_api.clj

```diff
@@ -52,14 +52,17 @@
       (doseq [user ["owner" "alice" "malice"]]
         (testing user
           (is (response-is-ok? (api-response :get "/api/catalogue/" nil
```

Comment:
this test should probably use some other API now that the catalogue can be public, for example `/api/my-applications/` should work